news/_posts/core-weekly/2019-02-11-coreweekly-week-06-2019.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 06 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-02-11 14:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 04th to Sunday 10th of February 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week, Pablo Borowicz, PrestaShop's current lead developer, started [a series on the blog about the past, present and future of the PrestaShop open source project](http://build.prestashop.com/news/prestashop-in-2019-and-beyond-introduction/). The next chapter is currently under review and should be published this week.
```

Comment:
```
Last week, Pablo Borowicz, PrestaShop's current lead developer, started [a series on the blog about the past, present and future of the PrestaShop open source project](http://build.prestashop.com/news/prestashop-in-2019-and-beyond-introduction/). The next chapter is currently under review and should be published this week.
```

Not sure about the "current", it may imply that that changes often ;)

"past, present and future of the PrestaShop open source project" --> wrong title

Reply:
:)

Reply:
done